lib/jobs/redfish-job.js

```diff
@@ -267,12 +269,75 @@ function redfishJobFactory(
             return data;
         });
     };
+
+    /**
+     * Check fabric service status and generate alert on status change
+     * @param redfish client utility
+     */
+    RedfishJob.prototype.fabricServiceDataAlert = function(redfish, currentData) {
```

Comment:
I think you need to rebase your branch, you're pushing commits already in master.
https://github.com/RackHD/on-tasks/blob/master/lib/jobs/redfish-job.js#L277